CHANGELOG.md

```diff
@@ -3,6 +3,175 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.0.0-alpha.0
+
+_Sep 22, 2022_
+
```

Comment:
We are missing a small introduction about the alpha, if you have some ideas @joserodolfofreitas

Reply:
Sorry I was too late to answer this.
But we can give a good introduction with the blog post:
https://deploy-preview-34424--material-ui.netlify.app/blog/mui-x-v6-alpha-zero/

Reply:
I'll do the release without it then and we will link the blogpost in the changelog when ready :+1: